doc/examples/reconst_dki.py

```diff
@@ -228,16 +186,61 @@
    **Diffusion tensor measures obtain from the diffusion tensor estimated from
    DKI (upper panels) and DTI (lower panels).**.
 
-From the figure, we can see that the DT standard diffusion measures from DKI
-are noisier than the DTI measurements. This is a well known pitfall of DKI
-[NetoHe2014]_. Since it involves the estimation of a larger number of
-parameters, DKI is more sensitive to noise than DTI. Nevertheless, DT diffusion
-based measures were shown to have better precision (i.e. less sensitive to
-bias) [Veraa2011]_.
+From the figure, we can see that the standard diffusion measures from DKI has
+similar constrasts than the standard diffusion measures from DTI.
+
+In addition to the standard diffusion statistics, the DiffusinKurtosisFit
+instance can be used to estimate standard non-Gaussian measures as the mean
+kurtosis (MK), the axial kurtosis (AK) and the radial kurtosis (RK).
+"""
+
+MK = dkifit.mk
+AK = dkifit.ak
+RK = dkifit.rk
+
+"""
+Kurtosis measures are susceptible to high amplitude outliers which can corrupt
+the automatic ajusted scale of the matplotlib functions. By assuming that
+typical values of kurtosis lie on a range between 0 and 2, the impact of high
+amplitude kurtosis outliers are removed in the following:
+"""
+
+MK[MK > 2] = 2
+MK[MK < 0] = 0
+AK[AK > 2] = 2
+AK[AK < 0] = 0
+RK[RK > 2] = 2
+RK[RK < 0] = 0
+
```

Comment:
I added this limits as an optional input on functions to compute the standard kurtosis measures. To add this directly on the procedures on DiffusionKurtosisFit instance, mk, ak and rk cannot anymore be defined as @auto_attr. To compute MK, AK and RK from the DiffusionKurtosisFit we have now to type:

MK = DiffusionKurtosisFit.mk()
AK = DiffusionKurtosisFit.ak()
RK = DiffusionKurtosisFit.rk()

Is this ok?